graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DefaultDgsFederationResolverTest.kt

```diff
@@ -124,14 +129,11 @@ class DefaultDgsFederationResolverTest {
             val type = DefaultDgsFederationResolver(entityFetcherRegistry, Optional.of(dgsExceptionHandler))
                 .typeResolver()
                 .getType(
-                    TypeResolutionEnvironment(
```

Comment:
Is this a breaking change? i.e. Users will be affected it looks like?

Reply:
It is a breaking change for those using the `TypeResolutionEnvironment` in such a manner. We will have to document this in our release notes. The fix is straight forward, and to be honest, the code that leverages the `builder` is cleaner.